src/components/sections/Footer.section.tsx

```diff
@@ -11,15 +11,20 @@ function FooterSection() {
 
   return (
     <footer className="bg-midnight text-white">
-      <img className="-mt-2" src={HeroIllustration} alt="Hero Illustration" />
-      <div className="mt-[-5.5rem] flex items-center justify-between px-24">
+      <img 
+        style={{ width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto' }}
```

Comment:
```suggestion
        className="w-full block mx-auto"
```

Reply:
True!! I've replaced the inline styles with Tailwind CSS classes, tested and it looks good to me!